oak_functions/loader/src/main.rs

```diff
@@ -66,6 +60,10 @@ struct Config {
     ///
     /// See https://docs.rs/tokio/1.5.0/tokio/runtime/struct.Builder.html#method.worker_threads.
     worker_threads: Option<usize>,
+    /// PEM encoded X.509 certificate that signs TEE firmware key. This field must be specified.
```

Comment:
This is the path to the certificate file, not the literal certificate, right? Could you clarify in the name (e.g. _file or _path)

Reply:
Done.